pom.xml

```diff
@@ -67,6 +67,7 @@
         <module>caf-logging-common</module>
         <module>caf-logging-log4j2</module>
         <module>caf-logging-logback</module>
+        <module>caf-logging-logback-tomcat</module>
```

Comment:
Badly named - nothing in the module is specific to Tomcat

Reply:
This new module with custom ConsoleAppender added for mean to the tomcat Access log. Now the classes are renamed to reflect its usage.